### ISSUE TITLE
Content editors not displayed on document type editor for nested content property

### ISSUE THREAD

**Issue by djomen** [leekelleher/umbraco-nested-content#36]
Before i start - LOVE this package from what i've seen until it broke! Great job!!! :)

My issue is its stopped working...

What i did:

I added the package via NuGet and created a test doctype called InfoBlock containing:

Textstring
MultilineText
ImageCropper
ContentPicker

Then added the data type Info Blocks and linked to InfoBlock
Added a property to a page and set to InfoBlocks type

On going to the content node (was an existing page): All the properties rendered ok and allowed entering of content

When i output the content in a web page the ImageCropper didn't contain the src property but did have the coordinates for the focus point.

I wondered if an issue with 7.3 umbraco so I created an image cropper property on its own (rather than nested type ) and that renders out fine so appeared to be an issue with Nested Content.

I then went to create some new types for nested content and now nothing will display in the property pages for nested content when editing a content node.

I even started a fresh with a rollback of the db but im not getting anything. All i get is the add button and the label for Item 1 - and no properties.

Not sure whats gone wrong.

Any ideas?

Thanks
Damian

**Comment by djomen** [comment]
<img width="914" alt="screen shot 2015-10-07 at 23 29 56" src="https://cloud.githubusercontent.com/assets/6400036/10353055/834ef250-6d4b-11e5-839f-e43d81424353.png">

This is what i see when these are the properties:
<img width="929" alt="screen shot 2015-10-07 at 23 32 26" src="https://cloud.githubusercontent.com/assets/6400036/10353121/c5a4965a-6d4b-11e5-9514-cf717e9d343f.png">

and the data type:
<img width="913" alt="screen shot 2015-10-07 at 23 36 37" src="https://cloud.githubusercontent.com/assets/6400036/10353205/5b838758-6d4c-11e5-96ca-758d0b2d9329.png">

**Comment by djomen** [comment]
I have also tried with a basic text item doctype and I get the same issue.

**Comment by djomen** [comment]
Also appears the drag & drop to set sort order gets ignored so poss a js issue.

**Comment by kjac** [comment]
Hey @djomen. I can't say for sure what's wrong as I haven't tried Nested Content in 7.3 yet, but one thing's for certain: Image Cropper does not work with Nested Content (and neither does Upload - see #27). You'll have to use a media picker and add crops to your media instead.

Alternatively you could consider using Archetype. It supports Image Cropper and Upload. Just a heads up if you switch: There's a known 7.3 issue with RTE's in Archetype. It's been fixed and a new release is imminent, but it's not out yet.

**Comment by leekelleher** [comment]
@djomen As @kjac says, the ImageCropper doesn't play nicely with Nested Content.  That looks like the cause of the main problem.

As for using simpler doctypes and nothing else displaying, is the ImageCropper also on there?

Are you getting any JS errors in your browser's dev-tools console? It might give some hint to the underlying issue.

Thanks, Lee.

**Comment by djomen** [comment]
Hi - Thanks - Yeah i had seen the issue with the upload and cropper so had removed it but cant get even simple NC working now. Not getting any issues in the console apart from a browserlink error - but don't think this is related.

Btw - archetype just had a release to fix it for 7.3

**Comment by djomen** [comment]
No @leekelleher i have removed the cropper. I just created a basic type with just a textbox and getting the same behaviour as above.  I have tried removing and re-installing the package also and still the same.

Also no console errors apart from a browserlink error. Was going to disable browserlink and try again.

**Comment by djomen** [comment]
Just installed into a clean 7.3 install with nuget and getting same issue with only a textfield on the nested type.

**Comment by djomen** [comment]
Nailed it! 

If you don't put the properties on the nested content docType onto a tab  - i.e. if you leave on general properties - it just doesn't work and you get the behaviour above - no matter what tabs/aliases you put on the data type or end use doc type. Moved the properties onto a tab and voila! 

Is this a bug or have i not used it correctly?

**Comment by leekelleher** [comment]
@djomen Yes! The properties need to be on a tab, (and generic properties are ignored).  I didn't spot this on your screengrab, doh! :tired_face: 

This is intended behaviour, as a feature is that a single doctype can have multiple tabs and be used as multiple Nested Content blueprints.

We'll update our [developer's guide](https://github.com/leekelleher/umbraco-nested-content/blob/develop/docs/developers-guide.md) to make this clearer.

Sorry for any headaches caused by this.

**Comment by djomen** [comment]
No probs - was just glad to get it working. :)  maybe a UI hint would also be good.

**Comment by leekelleher** [comment]
:+1: (on UI hint) Thanks!

**Comment by richardterris** [comment]
I know this issue is closed, but I've just installed Nested Content for the first time (which is the work of genius by the way) and I am having this issue.

Now down to testing a doc type which contains only a single field, which is of Text Area editor type and I can't get the fields to render on the content node.
I'm on 7.4.2, and I'm not seeing any console errors.

The one difference I see between my version and the documentation is that in the data type I have created, I don't have a drop down for tabs, I have a text field for tab alias. In here I've tried various things but I suppose it's possible the tab alias is wrong.

R
